defaults/main.yml

```diff
@@ -39,3 +39,4 @@ osbs_build_json_dir: /usr/share/osbs
 osbs_sources_command: fedpkg sources
 osbs_vendor: Fedora Project
 osbs_nodeselector: ''
+osbs_auto_build_node_selector: 'auto_build=true'
```

Comment:
Let's have an empty value as default since that's the "safer" value.

Reply:
deleted